WoWsShipBuilder.Core/DataContainers/Ship/AntiAirDataContainer.cs

```diff
@@ -50,7 +50,7 @@ public bool IsExpanderOpen
         constantDamageBonus = modifiers.FindModifiers("areaDamageMultiplier").Aggregate(constantDamageBonus, (current, value) => current * (decimal)value);
 
         IEnumerable<float> constantDamageBonusModifiers = modifiers.FindModifiers("lastChanceReloadCoefficient");
-        constantDamageBonus = Math.Round(constantDamageBonusModifiers.Aggregate(constantDamageBonus, (current, arModifier) => current * (1 + ((decimal)arModifier / 100))), 2);
+        constantDamageBonus = constantDamageBonusModifiers.Aggregate(constantDamageBonus, (current, arModifier) => current * (1 + ((decimal)arModifier / 100)));
 
```

Comment:
Consider if it may be necessary to round this value in order to avoid floating-point errors